jme3-effects/src/main/java/com/jme3/water/WaterFilter.java

```diff
@@ -292,8 +303,20 @@ protected void cleanUpFilter(Renderer r) {
     @Override
     public void write(JmeExporter ex) throws IOException {
         super.write(ex);
+
         OutputCapsule oc = ex.getCapsule(this);
 
+        final Spatial reflectionScene = getReflectionScene();
+        final boolean needSaveReflectionScene = isNeedSaveReflectionScene();
+
+        oc.write(causticsTexture, "causticsTexture", null);
+        oc.write(heightTexture, "heightTexture", null);
+        oc.write(normalTexture, "normalTexture", null);
+        oc.write(foamTexture, "foamTexture", null);
+        oc.write(needSaveReflectionScene, "needSaveReflectionScene", false);
+        oc.write(needSaveReflectionScene ? reflectionScene : null, "reflectionScene",
```

Comment:
I'm sure that it doesn't break back compatibility, but I will recheck it again ;)